### DISCUSSION TITLE
SMS notification bug(?)/feature request

### DISCUSSION THREAD

**Discussion by VisitorSnowdrop** [0perationPrivacy/VoIP#114] [General]
Love the app so far but have issue: I have it set to make unique sound on SMS arrival, but that doesn't seem to work on my Pixel5a GrapheneOS (though visual flag DOES work; i checked permissions and volume settings, and tried with app open to various profiles). Bug? One workaround -new feature- would be if I could configure app to email incoming SMS to my dedicated Tutanota account, where audible notifications do work. I could do that now but only by foregoing VoipSuite entirely. Don't want to forego the app, as I still need to use it to send replies (so need to have the whole conversation in VoipSuite).  Just hoping to get audible notifications one way or another. Thanks!

**Comment by 0perationPrivacy** [comment]
As it is right now, this is a web app. Not a native app. So audible notifications are controlled by the OS and the Browser you're using. The App has no control over this. On mobile, the browser does not allow notifications. Only on desktop browsers have this feature which works fine.

Email feature works perfectly well. Check the email smtp settings and the discussion on the various providers to use. I get emails for every sms received. Then I just open the app shortcut and view/reply.
Working on a native app right now which should increase adoption and decrease dependence on a web browser.

**Reply by getsean** [reply]
I have yet to see any notifications on any browser for incoming calls. Is this a current feature or just a red dot notification for an SMS?

**Reply by 0perationPrivacy** [reply]
> I have yet to see any notifications on any browser for incoming calls. Is this a current feature or just a red dot notification for an SMS?

It's a proper browser notification. You have to enable them in firefox. It's in the wiki. It will be like a side notification popup.
https://github.com/0perationPrivacy/voip/wiki/Web-Application
https://github.com/0perationPrivacy/voip/wiki/Calling

**Reply by getsean** [reply]
Thanks for the quick response. I was able to get SMS notifications working on Firefox. Call notifications still not working. Should calls just work after enabling them from SMS prompting it?
